OpenDF-Describe-Image/src/Server/exceptions/FileNotFound.py

```diff
@@ -0,0 +1,14 @@
+from flask import jsonify
+
+class FileNotFound(Exception):
+
+    def __init__(self, payload=None):
+        Exception.__init__(self)
+        self.message = "File Not Found"
+        self.status_code = 404
+        self.payload = payload
+
+    def to_dict(self):
+        error = dict(self.payload or ())
+        error['message'] = self.message
```

Comment:
Let's send an Error Code also. Refer this best practice guide. http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#errors